random.html

```diff
@@ -32,8 +32,8 @@
     </div>
 
 
-    <script src="src/index.js"></script>
-    <script type="text/javascript" src="src/random.js"></script>
+    <script type="module" src="src/index.js"></script>
```

Comment:
If you don't use anything from index.js just remove it :)

Reply:
You are right, i must miss it. I change it :)